mp/src/game/client/momentum/ui/SettingsPanel/MomentumSettingsDialog.cpp

```diff
@@ -175,7 +177,7 @@ void CMomentumSettingsDialog::Activate()
 {
     BaseClass::Activate();
 
-    SetActivePanel(m_pInputPage);
```

Comment:
What happens if this is just not called when the convar is true?

Reply:
Nothing; it pulls the remembered tab up correctly, when the convar is false though it won't reset it back to `m_pInputPage`, should I just have it pass `m_pInputPage`?

Reply:
Yep. So what I was getting at is, the active panel is already remembered, but I added these so that it mimics the regular Source settings, which forced the first tab every time because it was destroyed and recreated.

So for this, you'll just want to check if it's not enabled, then set to input page. Otherwise, you set to active page and make sure it scrolls to the top upon showing the settings panel again.